MangoDiary/src/components/CalendarPage.vue

```diff
@@ -4,35 +4,42 @@
       <div>
         <img src="/images/logo.png" class="logo">
       </div>
-      <div>
-          <select v-model="$store.state.selectedYear" @change="updateCalendar()">
-            <option v-for="year in yearRange" :key="year" :value="year">{{ year }}</option>
-          </select>
-        </div>
-        <div>
-          <select v-model="$store.state.selectedMonth" @change="updateCalendar()">
-            <option v-for="month in 12" :key="month" :value="month">{{ $store.state.monthNames[month-1] }}</option>
-          </select>
+      <div class="select-month">
+        <div class="left-select-month">
+          <font-awesome-icon icon="chevron-left" class="last-month" @click="handleClickChangeMonth(-1)"/>
+          <div class="month-block">
+            <div class="month-name"><b>{{ this.$store.state.selectedMonth }}</b></div>
+            <div class="year-block">
+              <div class="year">{{ this.$store.state.selectedYear }}</div>
+              <div class="month">{{ this.$store.state.monthNames[this.$store.state.selectedMonth - 1] }}</div>
```

Comment:
using Optional chaining (?.) and Nullish coalescing operator (??)

[Check this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)